docs/source/installation/guide_complete/index.rst

```diff
@@ -0,0 +1,363 @@
+First, connect to PostgreSQL using ``psql``, the `native command line client for PostgreSQL <https://www.postgresql.org/docs/current/app-psql.html>`_:
+
+.. code-block:: console
+
+    psql -h <hostname> -U <username> -W
+
+If PostgreSQL is installed on the localhost, ``<hostname>`` can be replaced with ``localhost``, and the default ``<username>`` is ``postgres``.
+While possible to use the ``postgres`` default user for the AiiDA profile, it is recommended to create a custom user:
+
+.. code-block:: sql
+
+   CREATE USER aiida-user WITH PASSWORD '<password>';
+
+replacing ``<password>`` with a secure password.
+The name ``aiida-user`` is just an example name and can be customized.
+Note the selected username and password as they are needed when creating the profile later on.
+
+After the user has been created, create a database:
+
+.. code-block:: sql
+
+   CREATE DATABASE aiida-database OWNER aiida-user ENCODING 'UTF8' LC_COLLATE='en_US.UTF-8' LC_CTYPE='en_US.UTF-8';
+
+Again, the selected database name ``aiida-database`` is purely an example and can be customized.
+Make sure that the ``OWNER`` is set to the user that was created in the previous step.
+Next, grant all privileges on this database to the user:
+
+.. code-block:: sql
+
+   GRANT ALL PRIVILEGES ON DATABASE aiida-database to aiida-user;
+
+After the database has been created, the interactive ``psql`` shell can be closed.
+To test if the database was created successfully, run the following command:
+
+.. code-block:: console
+
+    psql -h <hostname> -d <database> -U <username> -W
+
+replacing ``<database>`` and ``<username>`` with the chosen names for the database and user in the previous steps, and providing the chosen password when prompted.
+
+Once the database has been created, create a profile with:
+
+.. code-block:: console
+
+    verdi profile setup core.psql_dos
+
+The options specific to the ``core.psql_dos`` storage plugin are:
+
+* ``--database-engine``   The engine to use to connect to the database.
+* ``--database-hostname`` The hostname of the PostgreSQL server.
+* ``--database-port``     The port of the PostgreSQL server.
+* ``--database-username`` The username with which to connect to the PostgreSQL server.
+* ``--database-password`` The password with which to connect to the PostgreSQL server.
+* ``--database-name``     The name of the database in the PostgreSQL server.
+* ``--repository-uri``    URI to the file repository.
+
+.. _installation:guide-complete:validate-installation:
+
+
+Validate installation
+=====================
+
+Once a profile has been created, validate that everything is correctly set up with:
+
+.. code-block:: console
+
+    verdi status
+
+The output should look something like the following::
+
+    ✔ version:     AiiDA v2.5.1
+    ✔ config:      /path/.aiida
+    ✔ profile:     profile-name
+    ✔ storage:     SqliteDosStorage[/path/.aiida/repository/profile-name]: open,
```

Comment:
```suggestion
    ✔ storage:     Storage for 'profile-name' [open] @ postgresql://aiida-profile-name:***@localhost:5432/aiida-profile-name / DiskObjectStoreRepository: 288fcc39940543a1b67f85db1eb8cb8c | /path/.aiida/repository/profile-name/container
```
This was still the output of a `core.sqlite_dos` profile.

Reply:
This is intentional. Note that this subsection is not specific to `core.psql_dos` but a generic after-setup step. So I think it is fine to keep the example of the `core.sqlite_dos`. I have anyway cleaned it a bit such that it fits within the width of the code block. There is no need for the exact connection string